packages/meditrak-server/README.md

```diff
@@ -11,56 +11,6 @@ Backend for the MediTrak health facility survey and mapping software
 - `yarn` (to install dependencies)
 - `yarn start-dev` to run in dev mode or `yarn start` to build and run production
 
-## Local database
-
-By default, the DB_URL in the .env file will point to the database on the dev server. However, this
-can easily become confusing if multiple meditrak-server instances are changing data, so it's nicer
-to run a local database.
-
-Meditrak server actually uses exactly the same db as web-config-server, so if you've already set
-that up, you don't need to do anything except change the .env DB_URL and DISABLE_SSL variables
-
-If you haven't yet set up the db "tupaia", do the following (and note that you won't need to do this
-again if you work on web-config-server):
-
-Set up postgres on your machine and create the database 'tupaia', with credentials matching those in
-the .env file.
-
-The project requires importing an initial database dump. This can be obtained from the `#latest-db` channel
-on Slack. Import the dump by running:
-
-```bash
-psql tupaia -U tupaia < tupaia_dump.sql
-```
-
-## Tests
-
-### Running the tests
-
-```bash
-  yarn test               # Runs all tests
-  yarn test -g ${pattern} # Runs tests filtering their file names by ${pattern}
-  yarn test:coverage      # Runs tests and displays project test coverage
-```
-
-### Filtering describe/it blocks
-
```

Comment:
The existing test documentation ended up being a bit like a framework documentation, which is probably out of scope for our READMEs. We should focus on our code specifics, devs can look up the rest online